src/dug/server.py

```diff
@@ -315,6 +275,253 @@ async def search_study(study_id: Optional[str] = None, study_name: Optional[str]
     }
 
 
+@APP.post('/concepts', tags=['v2.0'], response_model=ConceptsAPIResponse)
```

Comment:
How come `/concepts` was not updated to use `Search.search_elements` like the other endpoints?

Reply:
Note: updated this endpoint to use search_elements on the filtering PR, so changes will come later.